src/main/java/jenkins/scm/api/MethodUtils.java

```diff
@@ -53,6 +53,7 @@ static boolean isAbstract(Class<?> clazz, String methodName, Class<?>... paramet
      * Checks if the  method defined on the base type with the given arguments
      * are overridden in the given derived type.
      */
+    // TODO replace with core utility method once JENKINS-40250 is available in base version of Jenkins
```

Comment:
Probably suffices to update to 1.633+ (i.e., 1.642.x.) where JENKINS-30002 was fixed.